modules/nw-sriov-cfg-bond-interface-with-virtual-functions.adoc

```diff
@@ -0,0 +1,122 @@
+
+Bond-CNI can be created using SR-IOV virtual functions and placing them in the container network namespace.
+
+{product-title} only supports Bond-CNI using SR-IOV virtual functions. Physical interfaces are not supported.
+
+.Prerequisites
+
+* The SR-IOV operator must be installed and configured to obtain virtual functions in a container.
+* To configure SR-IOV interfaces, an SR-IOV network and policy must be created for each interface.
+* The SR-IOV operator creates a network attachment definition for each SR-IOV interface, based on the SR-IOV network and policy defined.
+
+== Creating a bond network attachment definition
+
+Now that the SR-IOV virtual functions are available, you can create a bond network attachment definition.
+
+[source,yaml]
+----
+apiVersion: "k8s.cni.cncf.io/v1"
+    kind: NetworkAttachmentDefinition
+    metadata:
+      name: bond-net1
+      namespace: demo
+    spec:
+      config: '{
+      "type": "bond", <1>
+      "cniVersion": "0.3.1",
+      "name": "bond-net1",
+      "ifname: "bond0" <2>
+      "mode": "active-backup",
+      "failOverMac": 1, <3>
+      "linksInContainer": true, <4>
+      "miimon": "100",
+      "mtu": 1500,
+      "links": [ <5>
+            {"name": "net1"},
+            {"name": "net2"}
+        ],
+      "ipam": {
+            "type": "host-local"
+            "subnet": "10.56.217.0/24"
+            "routes": [{
+            "dst": "0.0.0.0/0"
+            }],
+            "gateway": "10.56.217.1"
+        }
+      }'
+
+----
+
+<1> The type is `bond`.
+
+<2> The `ifname` attribute specifies the name of the bond interface.
+
+<3> The `failover` attribute is mandatory for active-backup mode.
+
+<4> The `linksInContainer=true` flag tells the Bond CNI that the interfaces required are to be found inside the container. By default Bond CNI looks for these interfaces on the host which does not work for integration with SRIOV/Multus.
+
+<5> The `links` section defines which interfaces will be used to create the bond. By default, Multus names the attached interfaces as: "net", plus a consecutive number, starting with one.
+
+== Creating a pod using a bond interface
+
+You can now test the setup by creating a pod using a bond interface.
+
+[source,yaml]
+----
+apiVersion: v1
+    kind: Pod
+    metadata:
+      name: bondpod1
+      namespace: demo
+      annotations: <1>
+        k8s.v1.cni.cncf.io/networks: demo/sriovnet1, demo/sriovnet2, demo/bond-net1 <2>
+    spec:
+      containers:
+      - name: podexample
+        image: quay.io/openshift/origin-network-interface-bond-cni:4.9.0
+        command: ["/bin/bash", "-c", "sleep INF"]
+
+----
+
+<1> Note the network annotation.
+
+<2> The network annotation contains two SR-IOV network attachments, and one bond network attachment. The bond attachment uses the two SR-IOV interfaces as bond slaves.
```

Comment:
In terms of conscious language, is there another term that can be used for `slaves`? CCS is making an active effort to avoid/remove problematic words such as `master` and `slave`.